convert/common.c

```diff
@@ -76,7 +76,7 @@ static inline int write_temp_super(int fd, struct btrfs_super_block *sb,
 }
 
 /*
- * Setup temporary superblock at cfg->super_bynter
+ * Setup temporary superblock at cfg->super_bytenr
```

Comment:
Pretty sure this is right...

Reply:
Indeed, refers to the member 'super_bytenr' in struct btrfs_fs_info